src/mongo/db/query/get_executor.cpp

```diff
@@ -1137,6 +1137,10 @@ bool getDistinctNodeIndex(const std::vector<IndexEntry>& indices,
         if (indices[i].multikey && isDottedField) {
             continue;
         }
+        // Skip indices where the first key is not field
```

Comment:
Nitpick: we try to have our comments end in a period so that it's clear that the comment is complete.